src/components/Login/Login.js

```diff
@@ -24,6 +24,9 @@ class Login extends Component {
       loaded: false,
       tokenValid: false,
     };
+  }
+
+  componentDidMount() {
```

Comment:
Sometimes, hasToken method makes a problem. So, I moved this method into in componentDidMound method.